### PR TITLE
try to freerun to fish for hipster/goth fights

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/dailies.ts [hunk 1]
```diff
@@ -541,3 +542,27 @@ export function gingerbreadPrepNoon(): void {
...
 }
+
+export function hipsterFishing(): void {
+  if (have($familiar`Mini-Hipster`)) {
+    useFamiliar($familiar`Mini-Hipster`);
+  } else if (have($familiar`Artistic Goth Kid`)) {
+    useFamiliar($familiar`Artistic Goth Kid`);
+  } else return;
+
+  while (findRun(false) && get("_gothKidFights") < 7) {
```
Comment (Rinn) [inline src/dailies.ts:553]:
```suggestion
  while (findRun(false) && get("_hipsterAdv") < 7) {
```

`_gothKidFights` is to track the pvp fights acquired, you want `_hipsterAdv`

File: src/dailies.ts [hunk 2]
```diff
@@ -557,7 +564,12 @@ export function hipsterFishing(): void {
     const runSource = findRun(false);
     if (!runSource) return;
     if (runSource.prepare) runSource.prepare();
-    freeFightOutfit([...(runSource.requirement ? [runSource.requirement] : [])]);
+    freeFightOutfit([
+      ...(runSource.requirement ? [runSource.requirement] : []),
+      ...(have($familiar`Mini-Hipster`)
+        ? [new Requirement([], { forceEquip: [bestHipsterEquip] })]
```
Comment (Rinn) [inline src/dailies.ts:570]: passing all 3 with a bonus may be better in case any of the equipment isn't available it will just handle it

Comment (horrible-little-slime) [reply src/dailies.ts]: You're right, that's way better, implemented.

**Comment by horrible-little-slime** [comment]
Tested and functioning as of 8/4

**Review by Phillammon — APPROVED** [review]

**Pull request merged**